code/aind_smartspim_classification/classification.py

```diff
@@ -151,74 +151,77 @@ def cell_classification(smartspim_config: dict, logger: logging.Logger):
     # get quality blocks using mask
     chunks = [int(np.ceil(x / chunk_step)) for x in signal_array.shape]
     good_blocks = utils.find_good_blocks(
-        mask_array, chunks, chunk_step, smartspim_config["mask_scale"] - smartspim_config['downsample']
+        mask_array,
+        chunks,
+        chunk_step,
+        smartspim_config["mask_scale"] - smartspim_config["downsample"],
     )
 
     rechunk_size = [axis * (chunk_step // axis) for axis in signal_array.chunksize]
     signal_array = signal_array.rechunk(tuple(rechunk_size))
+    background_array = background_array.rechunk(tuple(rechunk_size))
```

Comment:
Are we sticking with dask for the classification or we want to use the new package for large-scale prediction?

Reply:
I think we are living right now with dask. I have been testing the new large-scale predictions and have some things we should talk about before switching over but that is the longer term goal